testdata/scripts/node/db/help.txtar

```diff
@@ -9,13 +9,10 @@ USAGE:
    chainlink node db command [command options] [arguments...]
 
 COMMANDS:
-   reset             Drop, create and migrate database. Useful for setting up the database in order to run tests or resetting the dev database. WARNING: This will ERASE ALL DATA for the specified database, referred to by CL_DATABASE_URL env variable or by the Database.URL field in a secrets TOML config.
-   preparetest       Reset database and load fixtures.
-   version           Display the current database version.
-   status            Display the current database migration status.
-   migrate           Migrate the database to the latest version.
-   rollback          Roll back the database to a previous <version>. Rolls back a single migration if no version specified.
-   create-migration  Create a new migration.
+   version   Display the current database version.
+   status    Display the current database migration status.
+   migrate   Migrate the database to the latest version.
+   rollback  Roll back the database to a previous <version>. Rolls back a single migration if no version specified.
 
```

Comment:
Oh, I wonder if we need to pass a `--tags test` arg here for this to work right?

Reply:
Might be a side effect of using the chainlink binary built with `make chainlink-install`.  I opened a PR here which fixes this, allowing it to run the one built in the local dir... not seeing this problem happen on that branch:
https://github.com/smartcontractkit/chainlink/pull/9632